cmd/dep/ensure.go

```diff
@@ -304,27 +304,23 @@ func getProjectConstraint(arg string, sm gps.SourceManager) (gps.ProjectConstrai
 }
 
 func checkErrors(m map[string]pkgtree.PackageOrErr) error {
```

Comment:
I think I've added reasonable test cases, and I'll update the integration tests (which I hadn't noticed until recently) to actually inspect the output.